p2p/enode/localnode.go

```diff
@@ -182,30 +181,24 @@ func (ln *LocalNode) updateEndpoints() {
 	// Determine the endpoints.
 	newIP := ln.fallbackIP
 	newUDP := ln.fallbackUDP
-	if ip, port := predictAddr(ln.udpTrack); ip != nil {
-		newIP = ip
-		newUDP = port
-	}
 	if ln.staticIP != nil {
 		newIP = ln.staticIP
+	} else if ip, port := predictAddr(ln.udpTrack); ip != nil {
```

Comment:
Is it ok to use the static IP with the fallback UDP port? Shouldn't we add a StaticUDP parameter?

Reply:
It doesn't make sense because there is no way to set such a port via anything in geth.